docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -271,6 +271,32 @@ faster and more stable. Add the ``--headless`` option to enable this mode.
 
 .. _selenium.webdriver: https://github.com/SeleniumHQ/selenium/tree/trunk/py/selenium/webdriver
 
+For testing changes to the admin UI, the selenium tests can be run with the
+``--screenshots`` option enabled. Screenshots will be saved to the
+``tests/screenshots/`` directory.
+
+To define when screenshots should be taken during a selenium test, the test
+class must use the ``django.test.selenium.screenshot_cases`` decorator with a
+list of supported screenshot types (currently ``"desktop"``, ``"mobile"``,
+``"rtl"`` and ``"dark"``). It can then call
+``self.take_screenshot("screenshot-name")`` at the desired point to generate
+the screenshots. For example::
+
+    from django.test.selenium import SeleniumTestCase, screenshot_cases
+    from django.urls import reverse
+
+
+    class SeleniumTests(SeleniumTestCase):
+        @screenshot_cases(["desktop", "mobile", "rtl", "dark"])
+        def test_login_button_centered(self):
+            self.selenium.get(self.live_server_url + reverse("admin:login"))
+            self.take_screenshot("login")
+            ...
+
+
+This generates three screenshots of the login page - one for a desktop screen,
+one for a mobile screen and one for right to left languages.
```

Comment:
With the addition of `dark`, I think it's now **four**... 😂 

Maybe we can avoid specifying a number and just list the screenshots generated?